client/jetpack-cloud/sections/agency-dashboard/sites-overview/sites-dashboard-v2.tsx

```diff
@@ -62,15 +62,18 @@ export default function SitesDashboardV2() {
 		? selectedSiteLicenses.reduce( ( acc, { products } ) => acc + products.length, 0 )
 		: selectedLicenses?.length;
 
-	const filtersMap: AgencyDashboardFilterMap[] = [
-		{ filterType: 'all_issues', ref: 1 },
-		{ filterType: 'backup_failed', ref: 2 },
-		{ filterType: 'backup_warning', ref: 3 },
-		{ filterType: 'threats_found', ref: 4 },
-		{ filterType: 'site_disconnected', ref: 5 },
-		{ filterType: 'site_down', ref: 6 },
-		{ filterType: 'plugin_updates', ref: 7 },
-	];
+	const filtersMap = useMemo< AgencyDashboardFilterMap[] >(
```

Comment:
The useMemo is not needed here. We are not calculating anything; all are hardcoded values without dependencies.

Reply:
Since it's a hardcoded map, let's move it outside, above `export default function SitesDashboardV2() {`. This:

```
const filtersMap: AgencyDashboardFilterMap[] = [
	{ filterType: 'all_issues', ref: 1 },
	{ filterType: 'backup_failed', ref: 2 },
	{ filterType: 'backup_warning', ref: 3 },
	{ filterType: 'threats_found', ref: 4 },
	{ filterType: 'site_disconnected', ref: 5 },
	{ filterType: 'site_down', ref: 6 },
	{ filterType: 'plugin_updates', ref: 7 },
];

export default function SitesDashboardV2() {
```

Reply:
And after that, we could remove `filterMap` as dependency in the `useEffect`, here: `}, [ isLoading, isError, sitesViewState.filters, filtersMap ] );`

Reply:
In my pagination PR I added a comment to that line suggesting we don't need to add filtersMap. Bud adding useMemo does appear to prevent those rerenders. So does moving it outside like @cleacos suggests :) Anyway mentioning as you've got a few merge conflicts from my PR, one of which is the comment about filtersMap, so feel free to remove that as needed.